package.json

```diff
@@ -18,9 +18,9 @@
     "test:crossbrowser": "npm-run-all -scl playwright:install test:crossbrowser:e2e test:crossbrowser:report",
     "test:crossbrowser:e2e": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run-multiple --all --config src/test/cross-browser/saucelabs.conf.js --grep @CrossBrowser",
     "test:crossbrowser:report": "allure generate functional-output/cross-browser/reports/* -c -o functional-output/cross-browser/allure ",
-    "test:functional": "yarn playwright install && yarn test:pa11y && yarn test:functional:min ; yarn test:functional:report",
-    "test:fullfunctional": "yarn playwright install && yarn test:pa11y && yarn test:functional:all ; yarn test:functional:report",
-    "test:functional:min": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run-workers 1 parallel --config ./src/test/functional/codecept.conf.js --grep '^(?!.*@nightly)'",
+    "test:functional": "yarn playwright install && yarn test:pa11y && yarn test:functional:min && yarn test:functional:report",
+    "test:fullfunctional": "yarn playwright install && yarn test:pa11y && yarn test:functional:all && yarn test:functional:report",
+    "test:functional:min": "NODE_TLS_REJECT_UNAUTHORIZED=0 codeceptjs run-workers 1 parallel --config ./src/test/functional/codecept.conf.js --grep \"(?=.*)^(?!.*@Nightly)\"",
```

Comment:
I'd be inclined to rollback the original PR and work out what isn't working. We're still using npm-run-all in other setups in this file, so I'm not sure why we needed to remove it here

Reply:
Reverted the changes.

Reply:
I meant revert the PR https://github.com/hmcts/idam-user-dashboard/pull/773